cmd/tendermint/commands/run_node.go

```diff
@@ -35,27 +35,27 @@ var (
 func init() {
 
 	// configuration options
-	runNodeCmd.Flags().StringVar(&moniker, "moniker", config.GetString("moniker"),
```

Comment:
Config is a global variable within `commands`, so long as it has been initialized we should be able to retrieve values from it without knowing what path it was derived from. We can make sure that it is set beforehand by doing all the data flag processing in the `cobra.command.prerun` I believe.